core/math/vector2.cpp

```diff
@@ -33,48 +33,53 @@
 #include "core/math/vector2i.h"
 #include "core/string/ustring.h"
 
-real_t Vector2::angle() const {
```

Comment:
Please put the methods you've moved at the end to reduce the noise, the methods that haven't been changed shouldn't be in the diff